test/mocks/data/comment_data.dart

```diff
@@ -0,0 +1,34 @@
+import "dart:math";
+
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:proxima/models/database/comment/comment_data.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+
+class CommentDataGenerator {
+  static List<CommentData> generateCommentData(int count) {
+    return List.generate(count, (i) {
+      return CommentData(
+        content: "content_$i",
+        ownerId: UserIdFirestore(value: "owner_id_$i"),
+        publicationTime: Timestamp.fromMillisecondsSinceEpoch(1000 * i),
+        voteScore: i,
+      );
+    });
+  }
+
+  static CommentData createRandomCommentData({
+    String? content,
+    UserIdFirestore? ownerId,
+    Timestamp? publicationTime,
+    int? voteScore,
+  }) {
+    return CommentData(
+      content: content ?? "content_${Random().nextInt(100)}",
```

Comment:
Here we should either pass an instance of `Random()` to the `createRandomCommentData` or have an instantiable `CommentDataGenerator` class that takes an instance of `Random` as an argument (I personally prefer option 2).

Also note that any random testing should be seeded (for reproducibly reasons) so when creating the `Random` instance in testing, please make sure to provide a `const` seed to its constructor parameter.

Reply:
The same goes for `test/mocks/data/firestore_post.dart`, even though I just saw that it was not done before... It must have slipped through the review at the time.

Reply:
I agree.

Reply:
Nice, I have created the issue #203 for it :+1: